tox.ini

```diff
@@ -28,6 +28,7 @@ deps =
     isort
     black
     numpy==1.26.4
```

Comment:
@drvinceknight is there a reason we've fixed the versions here?

Reply:
It was done here to avoid issues with 2.0: https://github.com/Axelrod-Python/Axelrod/pull/1446